### ISSUE TITLE
Feature request: print command text to console, but do not execute

### ISSUE THREAD

**Issue by capc0** [EthanSK/restore-terminals-vscode#11]
the idea is that the default command text is printed to the console, but has to be confirmed/executed by pressing enter within the terminal

**Comment by EthanSK** [comment]
can you find a vscode api function to do this? If so then I can implement

**Comment by capc0** [comment]
yes, the ```sendText``` has an optional boolean parameter which basically is the ```Enter```  press. Defaults to ```true```.

https://code.visualstudio.com/api/references/vscode-api#Terminal

from the docs

```
addNewLine?

Whether to add a new line to the text being sent, this is normally required to run a command in the terminal. The character(s) added are \n or \r\n depending on the platform. This defaults to true

```

**Comment by EthanSK** [comment]
Thanks! It has been updated. Simple set `"shouldRunCommands": false` in the configuration for each splitTerminal object in the array.

**Comment by Neurolitatis** [comment]
Can you please add a feature to change the color of these icons in the terminal tabs?
